src/goto-symex/symex_assign.h

```diff
@@ -17,6 +17,7 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 
 class byte_extract_exprt;
 class expr_skeletont;
+class function_application_exprt;
```

Comment:
This looks like it belongs in the cpp file, since we haven't changed anything here